Makefile

```diff
@@ -30,7 +30,7 @@ spec:
 	cd scripts &&\
 		BUNDLE_GEMFILE=Gemfile bundle exec rspec
 	cd concourse/scripts &&\
-		go test
+		BUNDLE_GEMFILE=../../scripts/Gemfile bundle exec go test
```

Comment:
We tried this in the past and it slowed the test suite down considerably. There's some context in:

- with Bundler: 8b403df1a1f5b422d6466ce3d2f3a1ccc79b8b5f
- without Bundler: ebca6530aa1917dcac8f17c9fd716ce4891b2587

Can you add it to the `exec.Command()` within the individual test instead?